newIDE/app/src/JsExtensionsLoader/LocalJsExtensionsLoader.js

```diff
@@ -79,6 +79,15 @@ module.exports = function makeExtensionsLoader(
                   );
                 }
 
+                // Load any cache clearing methods, if we have somewhere where
+                // to register them.
```

Comment:
you only copied the other messages, but the comment is a bit weird.
May I suggest updating it for others too to: `Load & register any ...`